README.md

```diff
@@ -51,3 +51,7 @@ To restart the data to init state use:
 If you have some problems with removing old containers use:
 
 `yes | docker-compose rm`
+
+### POSTMAN
+
+import file camp-trello-clone.postman_collection to postman
```

Comment:
I would love to see how to do it, or get some link to docs how to work with it (import, export) to narrow the time to do it